pkg/operator/tpr.go

```diff
@@ -59,7 +59,7 @@ func (o *Operator) createTPR() error {
 		}
 	}
 
-	return err
```

Comment:
we should revert this back. If we get an error that is not IsAlreadyExist, that will not be returned.
And for the happy path, err==nil so it will still work.